src/scripts/main.js

```diff
@@ -1,3 +1,38 @@
 'use strict';
 
-// write code here
+const employeesList = [...document.querySelectorAll('li')];
+
+function sortList(list) {
+  list.sort((a, b) => b.salary - a.salary);
+}
```

Comment:
either call it `sortBySalary` or remove this function and just create a callback with salary comparison 😉

Reply:
```Create function sortList, which takes a list property, and sorts list items in descending order by salary from data attribute. (Salary from data attribute is a string, do not forget to convert it to number, write helper function for it). Append sorted items to list.```